doc/docs/Scheme_User_Interface.md

```diff
@@ -359,11 +359,35 @@ The lengths of the block edges along each of its three axes. Not really a 3-vect
 —
 The directions of the axes of the block; the lengths of these vectors are ignored. Must be linearly independent. They default to the three lattice directions.
 
-**`ellipsoid`**
+### ellipsoid
 
 An ellipsoid. This is actually a subclass of `block`, and inherits all the same properties, but defines an ellipsoid inscribed inside the block.
 
-Here are some examples of geometric objects created using the above classes:
+### prism
+
+Polygonal prism type.
+
+**`vertices` [list of `vector3`]**
+—
+The vertices that make up the prism. They must lie in a plane that's perpendicular to the `axis`. Note that infinite (`infinity`) prism lengths are not supported. To simulate infinite geometry, just extend the edge of the prism a little past the cell.
+
+**`height` [`number`]**
+—
+The prism thickness, extruded in the direction of `axis`. `mp.inf` can be used for infinite height.
+
+**`axis` [`vector3`]**
+—
+The axis perpendicular to the prism. Defaults to `(0,0,1)`.
+
+**`center` [`vector3`]**
+—
+If `center` is *not* specified, then the coordinates of the `vertices` define the *bottom* of the prism
```

Comment:
I don't think this is true until we add the equivalent of #441 for scheme. I guess that can be accomplished via `define-post-processed-property`? The API will be a bit different though because `center` is a required argument in scheme (inherited from `geometric-object`).

Reply:
Thanks for pointing this out. I have removed the reference to `center` in the Scheme documentation. It would be good to add support for this in order to be consistent with the Python interface.